src/test/scala/com/redislabs/provider/redis/df/DefaultTestDataset.scala

```diff
@@ -0,0 +1,45 @@
+package com.redislabs.provider.redis.df
+
+import com.redislabs.provider.redis.rdd.Person.data
+import com.redislabs.provider.redis.rdd.{Person, SparkRedisSuite}
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.redis.RedisFormat
+import org.scalatest.Matchers
+
+/**
+  * @author The Viet Nguyen
+  */
+trait DefaultTestDataset extends SparkRedisSuite with Matchers {
+
+  import TestSqlImplicits._
+
+  lazy val expectedDf: DataFrame = Person.df(spark)
+
+  def populateDf(tableName: String): Unit = {
+    val df = spark.createDataFrame(data)
+    df.write.format(RedisFormat).save(tableName)
+  }
+
+  def createTempView(tableName: String): Unit = {
+    spark.createDataFrame(data).createOrReplaceTempView(tableName)
+  }
+
+  def verifyDf(actualDf: DataFrame): Unit = {
```

Comment:
Can we use this function to eliminate duplicated code in HashRedisDataframeSuite and RedisSqlStandaloneSuite as well?